internal/resourcecontrol/resource_control.go

```diff
@@ -48,7 +48,7 @@ func MakeRequestInfo(req *tikvrpc.Request) *RequestInfo {
 		writeBytes += int64(unsafe.Sizeof(r.Keys))
 	}
 
-	return &RequestInfo{writeBytes: writeBytes}
+	return &RequestInfo{writeBytes: writeBytes * req.ReplicaNumber}
```

Comment:
It makes things complicated. How about the deployment of cluster with witness node?

Reply:
On the other hand, customer already pay for the multiple storage replicas.